src/runtime/http.rs

```diff
@@ -1,17 +1,17 @@
-#[cfg(feature = "aws-auth")]
+#[cfg(any(feature = "aws-auth", feature = "in-use-encryption-unstable"))]
```

Comment:
I assume you're planning to switch this over to the setup I used for Azure in the merge?

Reply:
oh yes I left a review comment for this but forgot to post it :) yep, I was holding off on updating this since it'll need refactoring during the rebase anyway